pyteal.ini

```diff
@@ -0,0 +1,4 @@
+[pyteal]
+
+[pyteal-source-mapper]
+enabled = True
```

Comment:
TODO: the compiler is broken when `enabled = False`. So this needs to be detected on the fly when compiling.